vue.config.js

```diff
@@ -25,32 +25,26 @@ module.exports = {
   chainWebpack: (config) => {
     config.plugin('define').tap((args) => {
       const env = args[0]['process.env'];
-      let keys;
-      let keysFinance;
-
-      try {
-        keys = require('./keys-whitelist.json');
-      } catch {
-        keys = {
-          mainnet: [],
-          testnet: []
-        }
-      }
+      const dataPlaceholder = {
+        testnet: [],
+        mainnet: []
+      };
+      let configFile;
 
       try {
-        keysFinance = require('./keys-finance.json');
+        configFile = require('./keys-whitelist.json');
       } catch {
-        keysFinance = {
-          mainnet: [],
-          testnet: []
-        }
+        configFile = {
```

Comment:
redundant here, 
the catch should capture & throw the exceptions here

Reply:
> catch should capture
`console.error` added

> throw the exceptions here
Then it will be impossible to build the app without this json file